make/ide/netbeans/langtools/build.xml

```diff
@@ -242,7 +242,7 @@
                 </nbjpdastart>
                 <property
                     name="@{jpda.jvmargs.property}"
-                    value="-Xdebug -Xnoagent -Djava.compiler=none -Xrunjdwp:transport=dt_socket,address=${jpda.address}"
+                    value="-Xdebug -Djava.compiler=none -Xrunjdwp:transport=dt_socket,address=${jpda.address}"
```

Comment:
Maybe -Djava.compiler=none should be removed too. I don't know if this project file is still used but I assume attempting to debug will print a warning due to this option.